.github/workflows/ciab.yaml

```diff
@@ -0,0 +1,170 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: CDN-in-a-Box CI
+
+on:
+  push:
+  create:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
```

Comment:
Can we get this to ignore things like `traffic_control/`, `docs/`, `blueprints/`, `experimental/` etc. using [the `paths-ignore` key](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestpaths) (or you could use `paths`, I don't really care it just seems easier to use `paths-ignore`)? We don't need to spend a half hour running CiaB for things that don't affect the actual CDN stack.

Reply:
Okay, ignoring things that don't affect the CDN stack in 02d428003.